verification/cbmc/proofs/skipCollection/Makefile

```diff
@@ -0,0 +1,26 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+
+PROOF_UID=skipCollection
+CHECK_FUNCTION_CONTRACTS = $(PROOF_UID)
+HARNESS_FILE= $(PROOF_UID)_harness
+
+CBMC_OBJECT_BITS = 10
```

Comment:
This seems like a lot.  What caused the increase from default?

Reply:
The instrumentation for assigns clause creates many more objects.

Reply:
@feliperodri, does @remi-delmas-3000 know about that?